PROTOCOL.md

```diff
@@ -424,6 +429,64 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests are sent with `REQUESTID`. `REQUESTID` is used to identify which
+action the request was intended for and is used to protect against race
+conditions with the cancel button. The first decision of a match has a
+`REQUESTID` of 1, and each successful decision increments the counter by 1.
+`REQUESTID` is optional.
+
+`|/team ORDER|REQUESTID`
+
+> Chooses the team order. Numbers not listed are displaced to the back by swapping
+> them with the number that took their place. For example `/team 25` sets the team
+> order from default to 253416.
+
+`|/move NUMBER TARGET|REQUESTID`
+
+> Requests to use the selected move `NUMBER` on `TARGET` Pokemon. `NUMBER` starts at
+> 1 and can be up to 12 (though usually 4), where `NUMBER` calls the move indexed at
+> `NUMBER - 1` in the Pokemon's set.
+>
+> Moves with `TARGET` specify which position they are trying to use the move on as a
+> number wherein the opposing Pokemon are positive integers counting up from `1`
+> starting on the right. Ally Pokemon targets are negative integers counting down from
+> `-1` starting on the left. `TARGET` is optional and only needs to be specified for
+> single target moves in doubles/triples formats.
+
+`|/switch NUMBER|REQUESTID`
+
+> Requests to switch the active Pokemon with the Pokemon at position `NUMBER` on the
+> team. In cases where a Pokemon is KO'd, their replacement is chosen with `/switch`.
```

Comment:
Sure, and mention that it must correspond to a non-active, non-fainted Pokemon (so `/switch 1` is nonsensical at all times, because the first Pokemon is always an active Pokemon)